ansible.cfg

```diff
@@ -4,4 +4,3 @@
 forks = 50
 host_key_checking = False
 remote_user = ec2-user
-private_key_file = aws_keys_pairs.pem
```

Comment:
user will need to add that in, right? 
should it be 
`private_key_file = <enter private ssh key>` ?

Reply:
@sallyom so with this we should just put a note in that if you are using a key that is not the default ssh key for your system define it here. Also, we probably need to hardcode it in to the ansible-playbook step for the terraform steps